tests/console/check_resume.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that "resume=" kernel parameter is in the list of default parameters on Sle15-SP2
+# installation on different from s390x architectures.
+
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    if (check_var('ARCH', 's390x')) {
+        assert_script_run("grep -v 'resume=' /proc/cmdline", fail_message => "resume parameter found in /proc/cmdline");
+    } elsif (check_var('ARCH', 'x86_64')) {
+        assert_script_run("grep 'resume=' /proc/cmdline", fail_message => "resume parameter not found in /proc/cmdline");
+    } else {
+        script_run("grep 'resume=' /proc/cmdline") && record_soft_failure("bsc#1159294 - No resume option for arm and ppc");
```

Comment:
FYI script_run can also take an parameter <output> to give a description here. LGTM however